osi_object.proto

```diff
@@ -835,67 +837,143 @@ message MovingObject
 
-            // Vehicle is a trailer.
+            // A trailer is a non-motorized vehicle designed for being towed by
+            // a motorized vehicle to carry goods, animals or people.
             //
-            // \note The vehicle can be, but doesn't need to be,
-            // attached to another vehicle.
+            // \note The vehicle can be, but doesn't need to be, attached to
+            // another vehicle.
             //
             // \image html OSI_TYPE_TRAILER.svg
             //
             TYPE_TRAILER = 9;
 
             // Vehicle is a motorbike or moped.
             //
+            // \note Deprecated differentiation, use TYPE_MOTORCYCLE instead
+            //
             TYPE_MOTORBIKE = 10;
 
-            // Vehicle is a bicycle (without motor and specific lights).
+            // A motorcycle is a motorized vehicle designed primarily for
+            // passenger transportation.
+            // Compared to a car, typically fewer passive safety features such
+            // as a full passenger cell are present.
+            // This category includes typical two-wheeled motorcycles, but also
+            // three-wheeled vehicles like motorcycles with side-cars or even
+            // trikes.
+            //
+            TYPE_MOTORCYCLE = 10;
+
+            // A bicycle is a human-powered or motor-assisted, pedal-driven
+            // vehicle.
+            // This category includes typical two-wheeled bicycles, but
+            // also cargo-bikes and alike with more than two wheels.
             //
             TYPE_BICYCLE = 11;
 
-            // Vehicle is a bus.
+            // A bus is a motorized vehicle designed to carry multiple
+            // passengers.
             //
             TYPE_BUS = 12;
 
-            // Vehicle is a tram.
+            // A tram is a vehicle designed for using rail infrastructure for
+            // the transport of passengers on rail infrastructure.
+            // The rail infrastructure may fully or partially overlap with
+            // public road infrastructure.
+            // In contrast to trains, trams do not have exclusive right-of-way.
+            // Often as a series of connected vehicles.
             //
             TYPE_TRAM = 13;
 
-            // Vehicle is a train.
+            // A train is a vehicle designed for the transport of passengers and
+            // freight on rail infrastructure. 
+            // The rail infrastructure for trains is mostly grade-separated from
+            // the public road infrastructure as trains have exclusive
+            // right-of-way.
+            // Therefore, in case crossings with the road infrastructure occur,
+            // the exclusive right-of-way is ensured, e.g. by railway barriers.
+            // Often as a series of connected vehicles.
             //
             TYPE_TRAIN = 14;
 
             // Vehicle is a wheelchair.
             //
             TYPE_WHEELCHAIR = 15;
```

Comment:
The Wheelchair was sorted into Micromobility Device as well in the OpenX definition. We have to make sure that we don't keep surplus definitions here, either. But again, we could discuss in the group if this is maybe prominent enough to spearate it into a dedicated type.

Reply:
The discussion result was to separate out both wheelchair and stand-up scooter types from micro-mobility device in the harmonized traffic participation specification and specifically state this in the description.

I adopted the changed descriptions from [here](https://code.asam.net/simulation/harmonization/trafficparticipants/traffic-participants-specification/-/commit/624d53287bdee12a6eb24e2ec5456524def293b3).